test/Microsoft.ML.Benchmarks/Microsoft.ML.Benchmarks.csproj

```diff
@@ -34,10 +36,14 @@
     <None Include="..\data\wikipedia-detox-250-line-data.tsv" Link="Input\wikipedia-detox-250-line-data.tsv">
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </None>
-    <None Condition="Exists('..\data\external\WikiDetoxAnnotated160kRows.tsv')"
-                  Include="..\data\external\WikiDetoxAnnotated160kRows.tsv"
-                  Link="external\WikiDetoxAnnotated160kRows.tsv">
+
+    <TlcResourceFile Update="@(TlcResourceFile)">
```

Comment:
>TlcResourceFile [](start = 31, length = 15)

call it something non TLC

Reply:
Agreed, but the reason I chose the name originally is because of the URL:

http://aka.ms/tlc-resources

Can we change this URL?  Or make a new aka.ms URL pointing to the same location with a different name?